main.py

```diff
@@ -2,12 +2,30 @@
 import pyautogui
 import sys
 import uvicorn
-from fastapi import FastAPI
+from fastapi import FastAPI, WebSocket
 from fastapi.staticfiles import StaticFiles
 from fastapi.responses import FileResponse
 
 app = FastAPI()
+pyautogui.FAILSAFE = False
```

Comment:
From the [official docs](https://pyautogui.readthedocs.io/en/latest/): 
> I HIGHLY RECOMMEND YOU DO NOT DISABLE THE FAILSAFE.

What behavior do we get if we don't use it? Can we write a custom logic instead of this?

Reply:
The failsafe cuts the mouse control when you move to the top right corner of the screen. I found this to be more of an annoyance than a safety measure.

> Error:  PyAutoGUI fail-safe triggered from mouse moving to a corner of the screen. To disable this fail-safe, set pyautogui.FAILSAFE to False. DISABLING FAIL-SAFE IS NOT RECOMMENDED.

Reply:
Got it. I will open an issue to find an alternate for it later. Let's also move it to the `__name__=="__main__"` block.